config/database.yml

```diff
@@ -0,0 +1,26 @@
+# SQLite version 3.x
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
+  adapter: postgresql
+  port: 5432
+  pool: 5
+  timeout: 5000
+
+development:
+  <<: *default
+  database: rails_mecha-assist_development
+
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  <<: *default
+  database: rails_mecha-assist_development
+
+production:
+  <<: *default
+  database: db/rails_mecha-assist_test
```

Comment:
Since your production is going to be in heroku, you should remove the configuration for production here, as it will be added when you deploy to heroku